administration/package.json

```diff
@@ -77,18 +80,19 @@
     "css-minimizer-webpack-plugin": "^5.0.1",
     "dotenv": "^16.3.1",
     "dotenv-expand": "^10.0.0",
-    "eslint": "^8.48.0",
+    "eslint": "^9.21.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-jest": "^28.2.0",
-    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jest": "^28.11.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
     "eslint-plugin-jsx-expressions": "^1.3.2",
     "eslint-plugin-prefer-arrow": "^1.2.3",
-    "eslint-plugin-react": "^7.34.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-unicorn": "^52.0.0",
+    "eslint-plugin-react": "^7.37.4",
+    "eslint-plugin-react-hooks": "^5.1.0",
+    "eslint-plugin-unicorn": "^57.0.0",
     "file-loader": "^6.2.0",
+    "globals": "^16.0.0",
```

Comment:
:question: Do we actually need this package?

Reply:
I checked again ... I don't think it's needed